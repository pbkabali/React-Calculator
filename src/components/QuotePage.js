const QuotePage = () => <p>This is a Joke</p>;

export default QuotePage;
