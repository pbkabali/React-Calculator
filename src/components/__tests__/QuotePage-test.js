import renderer from 'react-test-renderer';
import QuotePage from '../QuotePage';

describe('QuotePage', () => {
  it('renders correctly', () => {
    const page = renderer.create(<QuotePage />);
    expect(page).toMatchSnapshot();
  });
});
