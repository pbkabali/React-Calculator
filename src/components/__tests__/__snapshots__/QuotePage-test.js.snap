// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuotePage renders correctly 1`] = `
<div
  className="side-padding home-container"
>
  <p>
    What did the triangle say to the circle?
    <strong>
       “You’re pointless.”
    </strong>
  </p>
</div>
`;
